src/theVoice/index.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+import { space, font } from '_utils/branding'
+
+import TheVoice from './TheVoice'
+
+const StyledTheVoice = styled(TheVoice)`
+  &.rm-small {
```

Comment:
Is it also suppose to include the Kairos CSS for the Voice component ?

```
.theVoice {
  padding: var(--space-xl) 0;
  font-weight: 500;
}
@media (--large) {
  .theVoice {
    padding: 48px 0;
    text-align: center;
  }
}
```

Reply:
This kairos CSS will go away when I integrate TheVoice in kairos.